docs/database_example.ipynb

```diff
@@ -471,8 +471,16 @@
    "name": "python3"
   },
   "language_info": {
+   "codemirror_mode": {
```

Comment:
this is a bit tedious, but could you please modify this notebook by hand to remove this diff?

Reply:
should be fixed